web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -424,6 +428,37 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 					}
 				}
 			}
+			try {
+				List<String> locales = new ArrayList<String>();
+				locales.add("");
+				locales.add("_de");
+				locales.add("_en_GB");
+				locales.add("_es");
+				locales.add("_fa");
+				locales.add("_fr");
+				locales.add("_hi");
+				locales.add("_in_ID");
+				locales.add("_it");
+				locales.add("_kh");
+				locales.add("_pt");
+				locales.add("_si");
+				locales.add("_zh_CN");
+				locales.add("_ru");
```

Comment:
They are hard coded because I can see no easy way to retrieve all the locales needed.